testing/nightly/cluster.sh

```diff
@@ -27,12 +30,8 @@ case ${action:-} in
         cluster_upgrade
         exit 0
         ;;
-    -*)
-        echo "FATAL: Unknown option: ${action}"
-        exit 1
-        ;;
     *)
-        echo "FATAL: Nothing to do ..."
+        echo "FATAL: Nothing to do ... (action='$action')"
```

Comment:
Maybe change message to "Unsupported $action"

Reply:
fixed